enzyme/Enzyme/Enzyme.cpp

```diff
@@ -1789,9 +1824,14 @@ class EnzymeBase {
 #endif
     }
 
+#if LLVM_VERSION_MAJOR >= 16
     bool status = HandleAutoDiff(
```

Comment:
You need to have the bool status = HandleAutodiff( as well in the other version.

Reply:
thanks, fixed